docs/learn/court.md

```diff
@@ -6,8 +6,321 @@ title: Decentralized Court
+`delegations` list. This `delegations` list was specified inside the call
+argument of the extrinsic `delegate` by the delegator. There is one edge case to
+notify here. At the point of calling `delegate` the `delegations` list is
+checked, whether the specified account ids are actually valid jurors of the
+court system. If not, the `delegate` extrinsic fails. But at the time of the
+selection process, the `delegations` list could contain invalid account ids,
+which don’t represent proper jurors anymore (for example the juror exited the
+court system). For this reason only the actual contained jurors are taken into
+consideration of all `delegations`. If there are no valid jurors inside the
+`delegations` list, the delegator is removed from the court pool (error inside
+the code: `SelectionError::NoValidDelegatedJuror`).
+
+For delegations a vote weight (code reference: `SelectionAdd::DelegationWeight`)
+goes to a random and valid juror from the `delegations` list, but the delegator
+risks (`slashable` and code reference `SelectionAdd::DelegationStake`) the
+`MinJurorStake` associated to the vote weight. If the juror makes bad decisions,
+the delegator loses the selected `MinJurorStake`.
+
+### Voting at predefined Time Points
+
+Jurors are requested to vote in a periodic interval (`RequestInterval`) at a
+known request block in the future. This ensures that jurors only need to check
+at predefined times if they need to take action. If there wasn’t this concept of
+predefined requests, the jurors would have needed to check in a much smaller
+time interval if they are selected in court cases.
+
+### Commitment Voting
+
+The voting system uses a commit-reveal scheme, which is required to prevent
+jurors from simply voting for the obvious plurality decision. There are voting,
+aggregation, and appeal phases. During the voting phase, jurors cast their votes
+as encrypted hashes (extrinsic `vote`), which must later be revealed (extrinsic
+`reveal_vote`) as raw information. If a juror fails to vote or reveal their
+vote, they lose their stake for that specific court case. The encrypted hash
+consists of a BlakeTwo256 hash of the juror account, the vote item (outcome),
+and a salt. The salt is a hash derived from the juror's signature of the
+specific court ID. Without a salt, a malicious actor could try every vote item
+to obtain raw information. If a juror's salt is known before the voting phase
+ends, they could be exposed and penalized (extrinsic `denounce_vote`) by those
+aware of the salt. This adds a layer of protection against cheating and
+increases trust in the system.
+
+### Appeals
+
+After the aggregation phase ends and all jurors have had the chance to make
+their votes public, anyone can appeal the plurality decision (during the appeal
+period). This triggers a new court round with more requested jurors and
+delegators' stakes or (in case after the last round) allows a global dispute to
```

Comment:
```suggestion
period). This triggers a new court round with more requested jurors and total stake or (in case after the last round) allows a global dispute to
```

Reply:
Replacing `delegator's` by `total stake` doesn't really fit here. I renamed it to `court participants stakes`.